pkgs/objective_c/tool/extra_methods.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Extra methods inserted into NSString by generate_code.dart.
```

Comment:
This is more like a data file maybe move it into tool/data or name it generate_code_data.dart

Reply:
Done